### PR TITLE
Add seq to jerk control input

### PR DESCRIPTION
**Pull request by icsl-Jeon**
https://github.com/LARR-Planning/los-keeper/assets/30062474/560f8a48-4dfb-4170-95a3-d204aa846dfb

Pipeline tested !

### CODE REVIEW

File: los_keeper_ros2/launch/run.launch.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def generate_launch_description():
             output={'both': 'log'},
             remappings=[("~/state", "/los_keeper/los_server_node/state"),
                         ("~/points", "/los_keeper/los_server_node/points"),
-                        ("~/object_state_array", 
-                         "/los_keeper/los_server_node/object_state_array")]
+                        ("~/target_state_array",
```
Comment (icsl-Jeon) [inline los_keeper_ros2/launch/run.launch.py:28]: @YunwooLee94 
I had to change this

**Comment by icsl-Jeon** [comment]
@YunwooLee94 Please merge this if you are okay :)

**Review by YunwooLee94 — APPROVED** [review]
Currently, "jerk" is being used as a control input, but in future experiments, it seems like speed and position can be used as control inputs. Of course, I will be responsible for the code implementation.

**Comment by icsl-Jeon** [comment]
Okay! BTW, I was talking about conflict. Is it okay if we let `StatePoly()` for chasing trajectory result? 
Please see the conflict.

**Pull request merged**